setup.sh

```diff
@@ -0,0 +1,11 @@
+build_in() {
```

Comment:
Scripts go in the bin/ directory and don't contain the ".sh" extension as the script/language is really an implementation detail.  I have a different version of this script called "build_all" which I'll create a PR for - we should use that instead.

Style note - we don't use tabs in source code (bash included).